entity-server-api/src/main/java/org/terracotta/entity/ConcurrencyStrategy.java

```diff
@@ -38,20 +41,24 @@
   public final int UNIVERSAL_KEY = Integer.MIN_VALUE; 
```

Comment:
From discussion with @myronkscott looks like this key also has the property of trying to pick the shortest path. That would be a great property for Ehcache read operations for example. Can this be confirmed?